corehq/ex-submodules/pillowtop/dao/django.py

```diff
@@ -7,26 +7,35 @@ class DjangoDocumentStore(DocumentStore):
     """
     An implementation of the DocumentStore that uses the Django ORM.
     """
-
-    def __init__(self, model_class, doc_generator_fn, model_manager=None):
+    def __init__(self, model_class, doc_generator_fn=None, model_manager=None, id_field='pk'):
```

Comment:
If attributes like `model_class` and `id_field` were moved directly to properties, it may simplify some of the code and base classes. For example the SMS class could be:

```python
class SMSDocumentStore(DjangoDocumentStore):
    model = SMS
    id_field = 'couch_id'
```

not sure if we have a standard pattern for that, but I generally like doing this sort of change to simplify the `__init__` logic / remove the need to override init

Reply:
I started with `id_field` as a property then changed it - also wasn't sure which was better but you're right that it makes the subclasses cleaner. I'll do a follow up.

Reply:
I took a closer look at this and it doesn't really make it that much cleaner except for SMS